source/opt/inline_pass.h

```diff
@@ -169,6 +166,52 @@ class InlinePass : public Pass {
   std::unordered_set<uint32_t> funcs_called_from_continue_;
 
  private:
+  // DebugInlinedAtContext keeps data used for generating DebugInlinedAt
+  // instructions.
+  class DebugInlinedAtContext {
+   public:
+    DebugInlinedAtContext(const Instruction* line, const DebugScope& scope)
+        : call_instr_line(line), call_instr_scope(scope){};
+
+    void UpdateInlinedAtContext(uint32_t callee_instr_inlined_at,
+                                uint32_t chain_head_id) {
+      callee_inlined_at2chain[callee_instr_inlined_at] = chain_head_id;
+    }
+
+    const Instruction* FunctionCallLine() { return call_instr_line; }
+    const DebugScope& FunctionCallScope() { return call_instr_scope; }
+    uint32_t InlinedAtChainHead(uint32_t callee_instr_inlined_at) {
+      auto chain_itr = callee_inlined_at2chain.find(callee_instr_inlined_at);
+      if (chain_itr != callee_inlined_at2chain.end()) return chain_itr->second;
+      return kNoInlinedAt;
+    }
+
+   private:
+    const Instruction*
+        call_instr_line;  // Line instruction of function call instruction
```

Comment:
Move the comment to the line before the declaration.  It will look cleaner.  The same for the next line.